azure-quantum/setup.py

```diff
@@ -22,7 +22,7 @@
 # To make sure that local builds still work without the same environment
 # variables, we'll default to 0.0.1 as a development version.
 
-version = os.environ.get("PYTHON_VERSION", "0.0.1")
+version = os.environ.get("PYTHON_VERSION", "1.2.4")
```

Comment:
we don't actually need to set version here as the release pipeline will set the package version automatically